### PR TITLE
BLorient590

### PR DESCRIPTION
**Pull request by thea-m**
@MarcinKrawczuk

### CODE REVIEW

**Comment by thea-m** [comment]
(Can you just review BLorient590, I will look at the conflict in the other file later)

File: LondonBritishLibrary/orient/BLorient765.xml [hunk 1]
```diff
@@ -30,20 +30,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                             <idno>Wright 261</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
-                    <msContents>
-                        <summary/>
-
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient765.xml:39]: as always, I am in favour of the empty element, as it lets the overall summary be generated and printed

File: LondonBritishLibrary/orient/BLorient590.xml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient590" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Hymn called Ṭǝbaba Ṭabibān, Hymns to Angels, Saints and Martyrs
-                </title>
+                <title>Ṭabiba Ṭabibān, collection of salām-hymns</title>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient590.xml:8]: I would write: Collection of ...

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
Solved conflict

**Review by eu-genia — APPROVED** [review]

**Pull request merged**